src/components/Facility/DischargedPatientsList.tsx

```diff
@@ -52,16 +63,85 @@ const DischargedPatientsList = ({
     pathParams: { id: facility_external_id },
   });
 
-  const { qParams, updateQuery, advancedFilter, FilterBadges, updatePage } =
-    useFilters({
-      limit: 12,
-      cacheBlacklist: [
-        "name",
-        "patient_no",
-        "phone_number",
-        "emergency_phone_number",
-      ],
-    });
+  const {
+    qParams,
+    updateQuery,
+    advancedFilter,
+    FilterBadges,
+    resultsPerPage,
+    updatePage,
+  } = useFilters({
+    limit: 12,
+    cacheBlacklist: [
+      "name",
+      "patient_no",
+      "phone_number",
+      "emergency_phone_number",
+    ],
+  });
+
+  const params = {
+    page: qParams.page || 1,
+    limit: resultsPerPage,
+    name: qParams.name || undefined,
+    patient_no: qParams.patient_no || undefined,
+    phone_number: qParams.phone_number
+      ? parsePhoneNumber(qParams.phone_number)
+      : undefined,
+    emergency_phone_number: qParams.emergency_phone_number
+      ? parsePhoneNumber(qParams.emergency_phone_number)
+      : undefined,
+    local_body: qParams.lsgBody || undefined,
+    district: qParams.district || undefined,
+    offset: (qParams.page ? qParams.page - 1 : 0) * resultsPerPage,
+    created_date_before: qParams.created_date_before || undefined,
+    created_date_after: qParams.created_date_after || undefined,
+    modified_date_before: qParams.modified_date_before || undefined,
+    modified_date_after: qParams.modified_date_after || undefined,
+    ordering: qParams.ordering || undefined,
+    category: qParams.category || undefined,
+    gender: qParams.gender || undefined,
+    age_min: qParams.age_min || undefined,
+    age_max: qParams.age_max || undefined,
+    date_declared_positive_before:
+      qParams.date_declared_positive_before || undefined,
+    date_declared_positive_after:
+      qParams.date_declared_positive_after || undefined,
+    ration_card_category: qParams.ration_card_category || undefined,
+    last_consultation_medico_legal_case:
+      qParams.last_consultation_medico_legal_case || undefined,
+    last_consultation_encounter_date_before:
+      qParams.last_consultation_encounter_date_before || undefined,
+    last_consultation_encounter_date_after:
+      qParams.last_consultation_encounter_date_after || undefined,
+    last_consultation_discharge_date_before:
+      qParams.last_consultation_discharge_date_before || undefined,
+    last_consultation_discharge_date_after:
+      qParams.last_consultation_discharge_date_after || undefined,
+    last_consultation_admitted_bed_type_list:
+      qParams.last_consultation_admitted_bed_type_list || undefined,
+    last_consultation__consent_types:
+      qParams.last_consultation__consent_types || undefined,
+    last_consultation__new_discharge_reason:
+      qParams.last_consultation__new_discharge_reason || undefined,
+    last_consultation_current_bed__location:
+      qParams.last_consultation_current_bed__location || undefined,
```

Comment:
is this filter even present for discharged patients view?

Reply:
my Bad i overlooked on that